### ISSUE TITLE
Improve Accessibility of Search Results

### ISSUE THREAD

**Issue by joashrajin** [Automattic/pocket-casts-ios#964]
We have gotten a report that using Screen Reader on iOS is very difficult with the updated search functionality. 

As mentioned by the user:

>  Lately, I've been having difficulty navigating the search results when using Voice Over on my iPhone. In the past, I had no issues finding what I was looking for, but after the recent updates, the search results seem disorganized. I often end up in irrelevant episodes or sponsored links instead of the desired results. Even after typing in the name of the podcast, I spend an endless amount of time swiping right without success. I'm not sure what has changed, but it's become quite frustrating.

### Steps to reproduce the behavior

- Enable Screen Reader
- Try to search for a podcast
- Due to the scrolling, it is very difficult to understand what is going on

##### Tested on [iPhone 13], iOS [16.5], Pocket Casts [7.41]

6489986-zd-woothemes

**Comment by DGartmann** [comment]
I'm affected by this issue as a VoiceOver user. For me, the search results are almost impossible to use so hope it can be fixed soon. Thanks a lot.

**Comment by joashrajin** [comment]
+1 in 6649921-zd-a8c

> Using Voiceover, I cannot click the queue button on the now playing screen or search results on the redesigned search screen
